### ISSUE TITLE
g-link not recognizing ":to"

### ISSUE THREAD

**Issue by calebanthony** [gridsome/eslint-plugin-gridsome#16]
**Tell us about your environment**	
* **ESLint version:**	5.16.0
* **eslint-plugin-gridsome version:**	1.2.5
* **Node version:**	10.8.0

**Please show your full configuration:**

```json
const path = require('path');

module.exports = {
  root: true,
  env: {
    browser: true,
    node: true,
  },
  extends: [
    'airbnb-base',
    'plugin:vue/recommended',
    'plugin:gridsome/recommended'
  ],
  rules: {
    'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'off',
    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
    'vue/html-indent': ['error', 2],
    'vue/no-v-html': 'off'
  },
  parserOptions: {
    parser: 'babel-eslint',
  },
  settings: {
    'import/resolver': {
      alias: {
        map: [
          ['^~', path.resolve(__dirname, './src')]
        ],
        extensions: ['.js', '.vue']
      }
    }
  },
  plugins: [
    'gridsome'
  ]
};

```

**What did you do?**	

```vue
<g-link
  class="post-card__link"
  v-bind:to="post.path"
>
  Link
</g-link>
```

**What did you expect to happen?**
The rule should have passed without issue.

**What actually happened?**

```
33:7  warning  Expected '<g-link>' elements to have 'v-bind:to' or 'to'     gridsome/require-g-link-to
```

**Additional Comments**
I'm setting up ESLint on a clean install of the Gridsome starter blog. Whether I do `v-bind:to=""` or `:to=""` it throws the error. I've also tried commenting out both my other configurations I'm extending (AirBnB and vue/recommended) and it still shows the error.

It only appears to work when I just use the normal `to=""`, which isn't what I need.

**Comment by calebanthony** [comment]
I'm also seeing the same issue with the `g-image` component, it's not recognizing that it has a `src=""`

**Comment by tyankatsu0105** [comment]
@calebanthony 
Thank you for this issue.
To be honest,  I know this situation :_(
May be this is related to 'vue-eslint-parser' or 'eslint-plugin-vue'.
Please show me your `package.json` or repo!!

**Comment by BernardZhao** [comment]
Having a similar issue:

```
28:15  warning  Expected '<g-link>' elements to have 'v-bind:to' or 'to'  gridsome/require-g-link-to
```

Which can be found here:
```
<g-link
  v-for="(service, index) in services"
  :key="index"
  :to="service.route"
  class="navbar-item"
>
  {{ service.name }}
</g-link>
```

My .eslintrc.js:
```
module.exports = {
  root: true,
  parser: "vue-eslint-parser",
  parserOptions: { ecmaVersion: 6 },
  env: {
    browser: true,
    node: true,
    es6: true
  },
  extends: [
    "eslint:recommended",
    "plugin:vue/recommended",
    "plugin:gridsome/recommended",
    "prettier",
    "prettier/vue"
  ],
  rules: {
    "no-console": process.env.NODE_ENV === "production" ? "error" : "off",
    "no-debugger": process.env.NODE_ENV === "production" ? "error" : "off"
  }
};
```

**Comment by BernardZhao** [comment]
I think this may have to do with the fact that the `to` and `src` statements are not on the same line as `g-link`, as that is the only time this happens.

**Comment by tyankatsu0105** [comment]
@BernardZhao @calebanthony 
Thank you for your detail info and kind advice :)
I think that this bug has been caused by `vue-eslint-parser`'s version.
Could you try this ?
```
yarn add vue-eslint-parser -D
```

**Comment by BernardZhao** [comment]
Thanks @tyankatsu0105, this totally solved the issue!

**Comment by tyankatsu0105** [comment]
@BernardZhao 
That's a relief :)
I'll add this solution for README!!
